pytket/pytket/qasm/qasm.py

```diff
@@ -75,8 +75,10 @@
     "u2": OpType.U2,
     "u1": OpType.U1,
     "rx": OpType.Rx,
+    "rxx": OpType.XXPhase,
```

Comment:
Also need to add `rxx` and `rzz` to `included_gates[qelib1]` below.

Reply:
Done.